README.md

```diff
@@ -139,6 +139,36 @@ only really makes sense if you're calling Grover directly (and not via middlewar
 Grover.new('<some URI with basic authentication', username: 'the username', password: 'super secret').to_pdf
 ```
 
+#### Adding cookies
+If you need to set cookies to your new page instance, just pass them along like this (please note that the only required properties are `name` and `value`. Check [.setCookies](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetcookiecookies) docs):
+
+```ruby
+myCookies = [
+  { name: 'sign_username', value: 'any@any.com', domain: 'mydomain' },
+  { name: '_session_id', value: '9c014df0b699d8dc08d1c472f8cc594c', domain: 'mydomain' }
+]
+Grover.new('<some URI with cookies', cookies: myCookies).to_pdf
+```
+
+If, let's say, you wanted to just forward the cookies that you got on your original request, you can do something like:
```

Comment:
Again please be careful on line length, many code editors work best when the line is shorter. Also be sure to use a formal tone

```suggestion
If you need to forward the cookies from the original request, you could extract them as such:
```